spyderlib/otherplugins.py

```diff
@@ -8,52 +8,73 @@
 Spyder third-party plugins configuration management
 """
 
-import os
-import os.path as osp
 import sys
 import traceback
 
 # Local imports
 from spyderlib.utils import programs
+from spyderlib.baseconfig import get_conf_path
+from spyderlib.utils.external.path import path as Path
 
 
-# Calculate path to `spyderplugins` package, where Spyder looks for all 3rd
-# party plugin modules
-PLUGIN_PATH = None
-if programs.is_module_installed("spyderplugins"):
-    import spyderplugins
-    PLUGIN_PATH = osp.abspath(spyderplugins.__path__[0])

```

Comment:
@nodd this is where you could get all paths for a namespace, the __path__[0] is taking only the last imported... But that should contain also the local path were you are devloping.